### ISSUE TITLE
V0.11 Takes a Long Time to Find Peer With Subnet

### ISSUE THREAD

**Issue by rauljordan** [prysmaticlabs/prysm#5339]
Currently, v0.11 spends around 15 slots looking for peers with the right subnet upon launch, and then eventually connects and succeeds in the handshake. This seems way too long and should be near instant in a single, perfect network with 2 nodes.
```
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet.
[2020-04-07 14:48:04] DEBUG sync: No peers found subscribed to attestation gossip subnet with committee index 0. Searching network for peers subscribed to the subnet
```

**Comment by prestonvanloon** [comment]
Is this a discv5 issue?

**Comment by rauljordan** [comment]
No evidence yet, we just notice it takes a long time before they are found

**Comment by nisdas** [comment]
Closed by #5349
